diff --git a/node_modules/@react-native/gradle-plugin/settings.gradle.kts b/node_modules/@react-native/gradle-plugin/settings.gradle.kts
index 14f1e9c..7483484 100644
--- a/node_modules/@react-native/gradle-plugin/settings.gradle.kts
+++ b/node_modules/@react-native/gradle-plugin/settings.gradle.kts
@@ -13,8 +13,6 @@ pluginManagement {
   }
 }

-plugins { id("org.gradle.toolchains.foojay-resolver-convention").version("1.0.0") }
-
 include(
     ":react-native-gradle-plugin",
     ":settings-plugin",
